shibboleth/resources/prod/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE HTML>
```

Comment:
Do we still need this file?

Reply:
I've reworded it a bit, to be more professional. Normally a user should never see this but in case it somehow happens anyway, I think it might be a good idea to just leave it with a link to the application.